misc/test_installed_version.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -ex
+
+# Usage: misc/test_installed_version.sh [wheel] [python command]
+# Installs a version of mypy into a virtualenv and tests it.
+
+# A bunch of stuff about mypy's code organization and test setup makes
+# it annoying to test an installed version of mypy. If somebody has a
+# better way please let me know.
+
+TO_INSTALL="${1-.}"
+PYTHON="${2-python3}"
+VENV="$(mktemp -d --tmpdir mypy-test-venv.XXXXXXXXXX)"
+trap "rm -rf '$VENV'" EXIT
+
+"$PYTHON" -m virtualenv "$VENV"
```

Comment:
Why not use `venv` instead of `virtualenv`? `venv` is in the stdlib, `virtualenv` has to be installed from  PyPI.